src/shipit_static_analysis/shipit_static_analysis/lint.py

```diff
@@ -0,0 +1,128 @@
+# -*- coding: utf-8 -*-
+from shipit_static_analysis import Issue
+from cli_common.command import run
+from cli_common.log import get_logger
+import itertools
+import json
+import os
+
+logger = get_logger(__name__)
+
+ISSUE_MARKDOWN = '''
+## mozlint - {linter}
+
+- **Path**: {path}
+- **Level**: {level}
+- **Line**: {line}
```

Comment:
Nit: Please add the following fields, if they make sense (at least `publishable`):

- **In patch**: {in_patch}
- **Third Party**: {third_party}
- **Publishable**: {publishable}

Reply:
Do we need to detect third party files for linting issues ? I did not implement this on MozLintIssue

Reply:
Yes, because otherwise we'd block/spam code reviews of people updating third-party scripts (e.g. React) without them being able to fix the entire file.

Reply:
Ok, now mozlint issues check they are not in a third party file, using the same code as clang-tidy issues.
I'll push the build to staging once it passes the test.